astroquery/atomic/core.py

```diff
@@ -178,19 +185,15 @@ def query_object_async(self, wavelength_range=None, wavelength_type='',
                            show_auto_ionizing_transitions=None,
                            output_columns=('spec', 'type', 'conf',
                                            'term', 'angm', 'prob',
-                                           'ener')):
+                                           'ener'),
+                           cache=True,
+                           get_query_payload=False):
```

Comment:
Why do we have separate async and sync methods rather than relying on the decorator?

Reply:
this was written before we started using the decorator or was written by someone who was unaware of it, and I didn't feel like refactoring further when I was solving this issue.

Reply:
would you mind opening an issue for it? I could be picked up by a prospective intern as a relatively straightforward cleanup.